bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyDeviceProfile.java

```diff
@@ -236,8 +236,9 @@ public void initFromThingType(String name) {
 
         isSensor = isHT || isFlood || isDW || isSmoke || isGas || isButton || isUNI || isMotion || isSense || isTRV
                 || isWall;
-        hasBattery = isHT || isFlood || isDW || isSmoke || isButton || isMotion || isTRV;
-        alwaysOn = !hasBattery || isMotion || isSense; // true means: device is reachable all the time (no sleep mode)
+        hasBattery = isHT || isFlood || isDW || isSmoke || isButton || isMotion || isTRV || isBlu;
+        alwaysOn = !hasBattery || (isMotion && !isBlu) || isSense; // true means: device is reachable all the time (no
```

Comment:
Just to be sure: A battery-powered non-Bluetooth motion sensor is considered always on?

Reply:
yes, it has kind of a low power mode; normale sensors and BLU devices are really in deep sleep and can't be waked up by an application

Reply:
So a "normal" (non-Bluetooth) battery-powered motion sensor has a low power mode, but is still reachable? What do you mean by "normal sensors" then, from your answer this seems to exclude motion sensors?

Reply:
Shelly Motion is the only one with this low power mode, having a battery. Sensors like Flood, H&T as well as BLU devices goto into deep sleep and are not reachable